### PR TITLE
Design pattern - fixed button group

### PR DESCRIPTION
**Pull request by silentcloud**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [x] Rebase before creating a PR to keep commit history clear.
- [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: docs/pattern/table/demo/fixed-button-group-bottom.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+---
+
+用在表格行数很多时（一般多于 20 行），又想对表格数据进行频繁的操作时，尤其适用在无限加载的表格中。
+
+```jsx
+import { Table, Dropdown, Button, Menu, Icon } from 'antd';
+
+const data = [];
+for (let i = 0; i < 100; i++) {
+  data.push({
```
Comment (superRaytin) [inline docs/pattern/table/demo/fixed-button-group-bottom.md]: data 应该用 let 定义吧。。

**Comment by afc163** [comment]
应该用 Affix 来做？

File: docs/pattern/table/demo/fixed-button-group-bottom.md (hunk 1 above)
Comment (benjycui) [inline docs/pattern/table/demo/fixed-button-group-bottom.md]: `const` 只是引用不变，这种情况用 `let` 的话，ESLint 是会报 Error 的，不过现在还没开那个规则。

Comment (superRaytin) [inline docs/pattern/table/demo/fixed-button-group-bottom.md]: soga。

**Comment by silentcloud** [comment]
@afc163 之前是打算 Affix 的，但到时候放页面上之后，浏览页面会很奇怪，页面还有很多其他内容，要考虑这点么？

**Comment by afc163** [comment]
用 iframe 模式如何？

@benjycui

**Comment by benjycui** [comment]
@silentcloud 试下 `iframe: true`，如果不行我再想其他办法。

image

**Pull request merged**